### PR TITLE
Remove Razor precompilation

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @ajaybhargavb

**Comment by pranavkm** [comment]
cc @NTaylorMullen

**Comment by ajaybhargavb** [comment]
Remove this line https://github.com/aspnet/Mvc/blob/prkrishn/precompilation/src/Microsoft.AspNet.Mvc.Razor/project.json#L16

**Comment by ajaybhargavb** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:shipit:

File: samples/MvcSubAreaSample.Web/MvcSubAreaSample.Web.xproj [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0.24720</VisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
   <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
-    <ProjectGuid>59e1be90-92c1-4d35-adcc-b69f49077c81</ProjectGuid>
-    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <ProjectGuid>45F6B3B6-D114-4D77-84D6-561B3957F341</ProjectGuid>
```
Comment (dougbu) [inline samples/MvcSubAreaSample.Web/MvcSubAreaSample.Web.xproj]: Minor: Odd to see the `<ProjectGuid>` change. Could you instead use one of the `.xproj` scripts? Most of them keep the GUID.

**Comment by ajaybhargavb** [comment]
Also need to remove https://github.com/aspnet/Entropy/tree/dev/samples/Mvc.Razor.PreCompilation

File: samples/MvcSubAreaSample.Web/MvcSubAreaSample.Web.xproj (hunk 1 above)
Comment (dougbu) [inline samples/MvcSubAreaSample.Web/MvcSubAreaSample.Web.xproj]: Then again, the GUID you're changing this _to_ seems to match what's already in `Mvc.sln`. Is this PR missing a commit?

File: Mvc.NoFun.sln [hunk 2]
```diff
@@ -99,7 +99,7 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "LocalizationSample.Web", "s
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "ActionConstraintSample.Web", "samples\ActionConstraintSample.Web\ActionConstraintSample.Web.xproj", "{EE0BD773-4D47-4AA8-8472-5A938A3953BA}"
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "MvcSubAreaSample.Web", "samples\MvcSubAreaSample.Web\MvcSubAreaSample.Web.xproj", "{E1D45BAF-3967-4D81-9217-22AA54941BF5}"
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "MvcSubAreaSample.Web", "samples\MvcSubAreaSample.Web\MvcSubAreaSample.Web.xproj", "{45F6B3B6-D114-4D77-84D6-561B3957F341}"
```
Comment (dougbu) [inline Mvc.NoFun.sln]: New GUID doesn't match what's in `Mvc.sln`

**Comment by dougbu** [comment]
:shipit: either moving `MvcSubAreaSample.Web` changes to another PR or after making the project GUID consistent in this PR.

**Pull request closed without merging**